aggregate/load_aggregated.py

```diff
@@ -69,3 +69,37 @@ def initialize_dataframe_geo_index(geography, columns=[]):
     rv = pd.DataFrame(index=indicies[geography], columns=columns)
     rv.index.rename(geography, inplace=True)
     return rv
+
+
+"""this is specifically to use for housing security and quality March 4th POP data"""
+def load_clean_pop_data(ind_name_str: str) -> pd.DataFrame:
+    """Function to merge the two files for the QOL outputs and do some standard renaming. Because
+    these are QOL indicators they remain in the same csv output with columns indicating year"""
+
+    read_excel_arg = {
+        "0812": {
+            "io": "./resources/ACS_PUMS/EDDT_ACS2008-2012.xlsx",
+            "sheet_name": "ACS08-12",
+            "usecols": "A:LO",
+            "dtype": {"Geog": str},
+
+        },
+        "1519": {
+            "io": "./resources/ACS_PUMS/EDDT_ACS2015-2019.xlsx",
+            "sheet_name": "ACS15-19",
+            "usecols": "A:LO",
+            "dtype": {"Geog": str},
+        },
+    }
+
+    df_0812 = pd.read_excel(**read_excel_arg["0812"])
+
+    df_1519 = pd.read_excel(**read_excel_arg["1519"])
+
+    df = pd.merge(df_0812, df_1519, on="Geog", how="left")
+
+    df = df.filter(regex=ind_name_str + "|Geog")
+
+    df.loc[df["Geog"] == "NYC", "Geog"] = "citywide"
+    
+    return df
```

Comment:
This is a proper design question for the group, I feel like some of our functions are being put into a few different python files that we should consider consolidating? For example, I have a `load_pums_2000` function in the `utils/dcp_population_excel_helpers` which now just loads the census data that erica sent us for the 2000 census pums - the code above seems to be very similar to that and is also coming form erica. I don't have an answer @td928 @SashaWeinstein @AmandaDoyle

Reply:
I reviewed the outputs on my local machine - everything looked good (col_names with proper tags and sequence, expected # of columns in each output, etc) except for the column order which I think needs to be implemented